add-domain

```diff
@@ -999,3 +999,32 @@ wiflix.xyz
 wiflix.yachts
 coronavirus.zone
 fullcopec.zone
+dexsceerner.net
+app.dexscreener-home.net
+dexscreener-home.net
+traolerjoe.org
+traderjoexyz.bylfg.org
+bylfg.org
+v2-o-p-e-n-s-e-a.com
+susni-swap.com
+sushi.swap-ether.site
+swap-ether.site
+open-sea.market-ntf.com
+market-ntf.com
+hyperilquid.org
+hyperilquid.xyz-trade.com
```

Comment:
Double of `xyz-trade.com`

- https://kb.mypdns.org/issue/MTX-127037/xyz-trade.com MTX-127037
- https://mypdns.youtrack.cloud/issue/MTXB-209555/xyz-trade.com MTXB-209555
- https://github.com/mypdns/matrix/issues/127037